pkg/rc/farm.go

```diff
@@ -210,8 +211,21 @@ START_LOOP:
 				// at this point the rc is ours, time to spin it up
 				rcLogger.NoFields().Infoln("Acquired lock on new replication controller, spawning")
 
+				// TODO: maybe we don't need to fetch the RC
+				// here, but that involves changing replication
+				// controller code which expands the scope of
+				// the change to watching RC keys rather than
+				// values.  Also it probably doesn't matter
+				// much since we won't actually be doing a
+				// fetch that often (only when lock not held)
+				rc, err := rcf.rcStore.Get(rcKey.ID)
```

Comment:
hmm. one might wonder what the effect will be on a bunch of small `Get`s - for example, what happens when we turn on the RC farms after an extended outage so nothing is locked, so eventually all the RCs will be gotten? (grammar?) Or what happens when one node releases all its locks?

I don't mind deferring this question until later.